sdk/ml/azure-ai-ml/tests/test_configs/automl_job/test_datasets/image_instance_segmentation/train/train_annotations.jsonl

```diff
@@ -0,0 +1,102 @@
4368738, 0.6471471471471472, 0.21042084168336672, 0.512012012012012, 0.20240480961923848, 0.4039039039039039, 0.18236472945891782, 0.2672672672672673, 0.1843687374749499, 0.24174174174174173, 0.18036072144288579, 0.16666666666666666, 0.18637274549098196, 0.15915915915915915, 0.19839679358717435, 0.15615615615615616, 0.3967935871743487, 0.16216216216216217, 0.4088176352705411, 0.16966966966966968, 0.40681362725450904, 0.21771771771771772, 0.4148296593186373, 0.24924924924924924, 0.4228456913827655, 0.2627627627627628, 0.4228456913827655, 0.5690690690690691, 0.342685370741483, 0.5690690690690691, 0.3346693386773547, 0.5720720720720721, 0.32064128256513025, 0.5855855855855856, 0.30861723446893785, 0.6246246246246246, 0.3066132264529058, 0.6486486486486487, 0.24048096192384769, 0.6576576576576577]]}, {"label": "milk_bottle", "bbox": "null", "isCrowd": 0, "polygon": [[0.6753507014028056, 0.7327327327327328, 0.6352705410821643, 0.7312312312312312, 0.6212424849699398, 0.7252252252252253, 0.5731462925851704, 0.7177177177177178, 0.5190380761523046, 0.7177177177177178, 0.5050100200400801, 0.7207207207207207, 0.46292585170340683, 0.7222222222222222, 0.43887775551102204, 0.7192192192192193, 0.3967935871743487, 0.7192192192192193, 0.3587174348697395, 0.7147147147147147, 0.3346693386773547, 0.7147147147147147, 0.3226452905811623, 0.7117117117117117, 0.312625250501002, 0.7012012012012012, 0.3066132264529058, 0.6876876876876877, 0.3046092184368738, 0.6636636636636637, 0.30861723446893785, 0.6306306306306306, 0.32064128256513025, 0.5960960960960962, 0.32064128256513025, 0.5885885885885885, 0.32665330661322645, 0.5795795795795796, 0.34468937875751504, 0.5690690690690691, 0.3787575150300601, 0.5690690690690691, 0.4969939879759519, 0.575075075075075, 0.5150300601202404, 0.5780780780780781, 0.531062124248497, 0.5840840840840841, 0.561122244488978, 0.5870870870870871, 0.6132264529058116, 0.5870870870870871, 0.6372745490981964, 0.5825825825825826, 0.6653306613226453, 0.5825825825825826, 0.7034068136272545, 0.5900900900900901, 0.7474949899799599, 0.6126126126126126, 0.7595190380761523, 0.6156156156156156, 0.781563126252505, 0.6156156156156156, 0.7995991983967936, 0.6081081081081081, 0.8096192384769539, 0.6066066066066066, 0.8156312625250501, 0.6081081081081081, 0.8296593186372746, 0.6186186186186187, 0.8637274549098196, 0.6201201201201201, 0.875751503006012, 0.6246246246246246, 0.8817635270541082, 0.6321321321321322, 0.8857715430861723, 0.6456456456456456, 0.8877755511022044, 0.6741741741741741, 0.8837675350701403, 0.6936936936936937, 0.8697394789579158, 0.7117117117117117, 0.8316633266533067, 0.7102102102102102, 0.8256513026052105, 0.7132132132132132, 0.8236472945891784, 0.7177177177177178, 0.8156312625250501, 0.7207207207207207, 0.8056112224448898, 0.7207207207207207, 0.7775551102204409, 0.7087087087087087, 0.7535070140280561, 0.7087087087087087, 0.7294589178356713, 0.7162162162162162, 0.7114228456913828, 0.7252252252252253, 0.6753507014028056, 0.7327327327327328]]}]}
```

Comment:
this test will start failing as soon as the ws gets deleted... we need to keep updating the path to image files with existing test ws... check out this commit - https://msdata.visualstudio.com/DefaultCollection/Vienna/_git/sdk-cli-v2/pullrequest/802884?_a=files&path=/tests/ml/automl_job/e2etests/test_automl_image_segmentation.py
"_update_jsonl_path"

Reply:
Makes sense Riti. Thanks for point out.